notebooks/227-whisper-subtitles-generation/README.md

```diff
@@ -13,7 +13,7 @@ In this notebook, you will use its capabilities for generation of subtitles for
 Notebook contains the following steps:
```

Comment:
Update the description that there are 2 tutorials: how to convert and how to optimize Whisper.

Reply:
Done